plugins/agents/process/process.go

```diff
@@ -0,0 +1,36 @@
+package process
+
+import (
+	"io/ioutil"
+	"strings"
+
+	"github.com/gansoi/gansoi/plugins"
+	"github.com/gansoi/gansoi/transports"
+	"github.com/pkg/errors"
```

Comment:
Gansoi has three groups of imports.

"standard lib"
"external packages"
"internal packages"

Can this be formatted like:

```go
import (
	"io/ioutil"
	"strings"

	"github.com/pkg/errors"

	"github.com/gansoi/gansoi/plugins"
	"github.com/gansoi/gansoi/transports"
)
```